app/controllers/lifestages_controller.rb

```diff
@@ -1,6 +1,28 @@
 class LifestagesController < FincapTemplatesController
+  attr_accessor :lifestage
+
   def resource
-    LifestageTemplate.new(Mas::Cms::Lifestage.find(params[:id]))
+    LifestageTemplate.new(lifestage)
   end
   helper_method :resource
+
+  def show
+    @lifestage = Mas::Cms::Lifestage.find(params[:id])
+
+    @latest_news = tagged_news_items(news_tags).map do |news_item|
+      NewsTemplate.new(news_item)
+    end
+  end
+
+  private
+
+  def tagged_news_items(tags)
+    Mas::Cms::News.all(params: { tag: tags }).reject do |news_item|
+      news_item.slug == lifestage.slug
+    end
+  end
+
+  def news_tags
+    lifestage.tags ? lifestage.tags : []
+  end
```

Comment:
Can we move this methods (tagged news items and new_tag to an object? They seem to be a criteria that is worth to have unit tests.